web/src/components/avatar/group-item/styles.ts

```diff
@@ -0,0 +1,61 @@
+import { JssStyle } from 'jss'
+import { createUseStyles } from 'react-jss'
+
+import tokens from '../../../tokens'
+import { parseToken } from '../../../utils'
+
+import { GroupItemProps } from '.'
+
+export default createUseStyles(
+  (theme: typeof tokens): Record<string, JssStyle> => ({
+    name: {
+      whiteSpace: 'nowrap',
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      textAlign: 'left',
+      flexShrink: 0,
+      maxWidth: ({ maxWidth, size }) => {
+        if (!maxWidth) return 'auto'
+
+        const padding = parseToken(theme.spaceAvatarItem) * 2
+
+        const avatarSize = parseToken(
+          size === 'small' ? theme.sizeAvatarSmall : theme.sizeAvatarMedium,
+        )
+
+        return `calc(${maxWidth}px - ${padding + avatarSize}px)`
+      },
+      color: ({ isInactive }: GroupItemProps) =>
+        isInactive
+          ? theme.colorTextAvatarItemPrimaryInactive
+          : theme.colorTextAvatarItemPrimary,
+    },
```

Comment:
This code is exactly the same as in avatar-item, same for description.
Can we just have one style for those?

For group description, there is an extra margin that could be inlined or extended. I would avoid classes that have the same style and same purpose for multiple variations!

Reply:
Yes, thanks. Missed that